src/libraries/System.Private.CoreLib/src/System/Diagnostics/Stopwatch.cs

```diff
@@ -89,6 +89,8 @@ public void Restart()
             _isRunning = true;
         }
 
+        public override string ToString() => Elapsed.ToString();
```

Comment:
Since Stopwatch isn't sealed, technically this should also be added to the ref:
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime/ref/System.Runtime.cs
Otherwise, a type that derives from it and calls base.ToString() might end up skipping this override.  And because it's publicly visible, it should also have XML comments added to it to seed the docs

Reply:
Thanks @stephentoub for the guidance. I have added the XML comments and updated the `System.Runtime.cs` file as well (for now I just copied what was being done with other classes that have a `ToString()` override, but later on, I plan to look into the whole `ref` mechanism to understand how it works).